core/commonMain/src/kotlinx/serialization/Serializers.kt

```diff
@@ -58,14 +58,14 @@ public inline fun <reified T> SerializersModule.serializer(): KSerializer<T> {
  * Creates a serializer for the given [type].
  * [type] argument is usually obtained with [typeOf] method.
  *
- * This overload works with full type information, including type arguments and nullability,
- * and is a recommended way to retrieve a serializer.
- * For example, `serializer<typeOf<List<String?>>>()` returns [KSerializer] that is able
```

Comment:
I don't know what this dependent-type-style syntax means and was assuming that this was a copy-paste artifact of some sort.

Reply:
I think I mistyped the function call: it should be `serializer(typeOf<List<String?>>())`. Can you please bring this paragraph back in the correct form?